vsts/generate_javadocs_branch.ps1

```diff
@@ -56,85 +63,101 @@ function CreateJavadocReleaseBranch(
     TestLastExitCode  # stop if we can't create the branch
 
     Write-Host "Copying generated javadocs to replace current javadocs"
-    Set-Location apidocs
+    Set-Location apidocs    
```

Comment:
extra spaces appended

Reply:
I'll have to make this fix in master. This is just a merge commit. Let me send out a PR and then merge again.

Reply:
Here is the PR - https://github.com/Azure/azure-iot-sdk-java/pull/1031